Tools/LogParser/README.md

```diff
@@ -0,0 +1,9 @@
+# LogParser
+
+A package for parsing the log output of the Ably Asset Tracking SDK and its example apps.
```

Comment:
Might be worth being more specific here, as I'm assuming this isn't compatible with Android SDK logs:

```suggestion
A package for parsing the log output of the Ably Asset Tracking Swift SDKs and their example apps.
```

Reply:
Fair point, updated it 👍